net.purrdata.PurrData.yml

```diff
@@ -0,0 +1,118 @@
+id: net.purrdata.PurrData
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-wrapper.sh
+finish-args: [
+    # Display
+    '--share=ipc', '--socket=x11', '--socket=wayland', '--device=dri',
+    # Audio access
+    '--socket=pulseaudio',
+    # Filesystem access (pd doesn't yet work with portals)
+    '--filesystem=host',
+    '--filesystem=home',
+    # MIDI access (Flatpak doesn't yet support exposing only MIDI devices)
+    '--device=all'
+]
+modules:
+- name: purr-data
+  sources:
+  - type: git
+    # This fork of https://github.com/agraef/purr-data/ simply removes the Gem
+    # submodule, as it took way too long to clone this submodule for me.
+    url: https://github.com/ssssam/purr-data
+    branch: sam/2.10.0-disable-gem
+    commit: 17903852ea5bc564ff809d286dbed5217ba461af
+  - type: patch
+    path: patch/build-api.patch
```

Comment:
Technically speaking you can just use `script` source instead of patch (but would be better to have it upstreamed, of course!)